### PR TITLE
 Init multi wallet in main.go from window.go

### LINKED ISSUE #786: GoDCR crashes after 10-15 minutes of inactivity
GoDcr crashes after a few minutes of inactivity, giving the error in the attached screenshot

Issue

Issue also occurs 5-10 minutes after syncing has commenced.

issue 2

### PR DESCRIPTION
**Pull request by AdimekweEbuka**
Fix #786

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -70,6 +70,12 @@ func main() {
 		return
 	}
 
+	newError := wal.InitMultiWallet()
```
Comment (dreacot) [inline main.go]:
```suggestion
	err = wal.InitMultiWallet()
```

**Review by devchoplife — COMMENTED** [review]
@AdimekweEbuka Please add details about what you did to resolve the issue

**Comment by AdimekweEbuka** [comment]
> @AdimekweEbuka Please add details about what you did to resolve the issue

`InitMultiWallet()` function was being called multiple times because of its location in the StageRunning event handler when it should be only called once. It was then moved to the main.go to ensure that it's only being called once.

**Comment by AdimekweEbuka** [comment]
> @AdimekweEbuka Please add details about what you did to resolve the issue
Capture2

This was from Wisdom in the Godcr room, when the issue was brought up. And moving it to the main function stopped the crash. 

Please assist in testing to see if it has also been rectified at your end.

**Review by Sirmorrison — APPROVED** [review]

**Pull request merged**